.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!-- Please adhere to the style guide at -->
+<!-- https://github.com/zeitgeistpm/zeitgeist/blob/main/docs/STYLE_GUIDE.md -->
+
+<!-- Include references to the issues it fixes here separated by whitespaces. -->
+<!-- Use a valid GitHub keyword for that, such as "closes" or "fixes". -->
+<!-- Example: closes #500 #700 -->
+
+### What does it do?
+
+### What important points reviewers should know?
+
+### Is there something left for follow-up PRs?
+
+### What alternative implementations were considered?
+
+### Are there relevant PRs or issues in other repositories (Substrate, Polkadot, Frontier, Cumulus)?
```

Comment:
This template was taken off of Moonbeam, right? I don't think having Frontier in here makes a lot of sense.